### DISCUSSION TITLE
How to assign initial heads to a 3D model?

### DISCUSSION THREAD

**Discussion by flydream0428** [timflow-org/timflow#22] [Q&A] [answered]
Can someone please tell me how to assign initial heads to a 3D transient model with TTIM?

**Comment by mbakker7** [comment, marked as answer]
Thank you for your question, @flydream0428.
TTim computes the head change caused by all the transient elements starting at time t=0. 
The head change at t=0 equals zero.
You can add any initially steady state flow field by adding a TimML model.
TTim cannot simulate an initial condition that is not steady.
I hope this helps.

**Reply by mbakker7** [reply]
Just to clarify: You can add a TimML model to a TTim model using the `timmlmodel` keyword when creating a `Model3D` or `ModelMaq`.

**Reply by flydream0428** [reply]
Thanks a lot @mbakker7 , it works!
